api/src/main/java/org/apache/iceberg/UpdateStatistics.java

```diff
@@ -27,9 +27,21 @@ public interface UpdateStatistics extends PendingUpdate<List<StatisticsFile>> {
    * the snapshot if any exists.
    *
    * @return this for method chaining
+   * @deprecated since 1.8.0, will be removed 1.9.0 or 2.0.0, use setStatistics(statisticsFile).
```

Comment:
sorry I missed this during the first review, but this should be using proper javadoc: `use {@link #setStatistics(StatisticsFile)}`

Reply:
fixed in latest commit